src/utils/send-it.js

```diff
@@ -12,29 +32,44 @@ const deduplicate = (localRawFileNames, existingRawFileNames) => {
   return uniqueRawFileNames;
 };
 
+const uploadUniqueFilesToS3 = async (bucketFolder, localRawFileNames) => {
+  const existingRawFileNames = await s3.listRawFileNames(bucketFolder);
+  const uniqueLocalRawFileNames = deduplicate(localRawFileNames, existingRawFileNames);
+  s3.uploadFolderContents(uniqueLocalRawFileNames, uploadsFolder, targetFolder);
+};
+
 const main = async () => {
   const localRawFileNames = fs.readdirSync(uploadsFolder);
   console.log(`Total number of files found from local folder [${uploadsFolder}]: ${localRawFileNames.length}`);
 
-  // TODO - Simply grabbing both the live and test folders for now.
-  // There is a WIP feature that will change this to be driven by user input.
-  const existingRawFileNames = await s3.listRawFileNames([
-    'staging/',
-    'tweeted/',
-    'quarantine/',
-    'test-staging/',
-    'test-tweeted/',
-    'test-quarantine/',
-  ]);
-  const uniqueLocalRawFileNames = deduplicate(localRawFileNames, existingRawFileNames);
-
-  await s3.uploadFolderContents(uniqueLocalRawFileNames, uploadsFolder, targetFolder);
+  if (!targetFolder) {
+    await inquirer
```

Comment:
According to the [docs](https://github.com/SBoudrias/Inquirer.js/#inquirerpromptquestions-answers---promise), ` inquirer.prompt(questions)` returns a `Promise`, so we can utilize `async/await`.  This should let this code finish before the `Done uploading files` log is hit.

Let me know if you'd like to pair on this!